docs/Analyzer Configuration.md

```diff
@@ -0,0 +1,46 @@
+# Analyzer Configuration
+
+Starting with version `2.6.3`, all the analyzer NuGet packages produced in this repo, including the FxCop Analyzers NuGet package, support _.editorconfig based analyzer configuration_. End users can configure the behavior of specific CA rule(s) OR all configurable CA rules by specifying supported key-value pair options in an `.editorconfig` file. You can read more about `.editorconfig` format [here](https://editorconfig.org/).
+
+## .editorconfig format
+Analyzer configuration options from an .editorconfig file are parsed into _general_ and _specific_ configuration options. General configuration enables configuring the behavior of all CA rules for which the provided option is valid. Specific configuration enables configuring each CA rule ID or CA rules belonging to each rule category, such as 'Naming', 'Design', 'Performance', etc. Our options are _case-insensitive_. Below are the supported formats:
+   1. General configuration option:
```

Comment:
so, this option is only for ones from roslyn-analyzer, or generic for any analyzer? asking since dotnet_code_quality naming looks like very generic.

Reply:
Its intended only for analyzers in this repo. I can make it more specific but this felt more readable to me.